playground/src/props.js

```diff
@@ -0,0 +1,104 @@
+import { watch } from 'vue'
+import {
+  children,
+  on,
+  ref,
+  template,
+  effect,
+  setText,
+  render as renderComponent // TODO:
+} from '@vue/vapor'
+
+export default {
+  props: undefined,
+
+  setup(_, {}) {
+    const count = ref(1)
+    const handleClick = () => {
+      count.value++
+    }
+
+    const __returned__ = { count, handleClick }
+
+    Object.defineProperty(__returned__, '__isScriptSetup', {
+      enumerable: false,
+      value: true
+    })
+
+    return __returned__
+  },
+
+  render(_ctx) {
+    const t0 = template('<button></button>')
```

Comment:
In this design, what I understand is that the DOM finally generated by the template is like this
`<button></button><Comp></Comp>` -> `<button></button><p>3 * 2 = 6</p>` 
This is a fragment, but how to deal with a structure like this? I don't seem to see similar logic in the code
`<button><Comp></Comp></button>` -> `<button><p>3 * 2 = 6</p></button>`

Reply:
Since `renderComponent` can take a container as its third argument, I think it can be achieved by passing the button's Node as the container. 🤔 

```ts
const t0 = template('<button></button>')
const n0 = t0()
const {
  0: [n1]
} = children(n0)
on(n1, 'click', _ctx.handleClick)

renderComponent(
  child,
  {
    get count() {
      return _ctx.count
    },
    get inlineDouble() {
      return _ctx.count * 2
    }
  },
  n1 // button node
)
```


https://github.com/vuejs/core-vapor/assets/71201308/520546b5-165f-4134-8407-2a7aebea9714



I've written TODO comments in the import statements, and I've mentioned this in the description of this PR as well, but since I'm using the existing render function somewhat forcibly, I'm thinking of implementing a function like `createComponent`.